charts/fybrik/templates/katalog-connector/katalog-connector-deployment.yaml

```diff
@@ -46,10 +46,25 @@ spec:
           resources:
             {{- toYaml .Values.katalogConnector.resources | nindent 12 }}
           env:
+            - name: SERVICE_PORT
+              value: {{ .Values.katalogConnector.service.port | quote }}
             - name: PRETTY_LOGGING
               value: {{ .Values.global.prettyLogging | quote }}
             - name: LOGGING_VERBOSITY
               value: {{ .Values.global.loggingVerbosity | quote }}
+            - name: CONNECTOR_USE_TLS
```

Comment:
these variables are part of the connector env variables, you don't need the "CONNECTOR" prefix